### PR TITLE
Fix(Serverless): Add Explanation Regarding Fragmented Distributed Traces

### PR DESCRIPTION
**Pull request by kolanos**
We've had customers ask about this. Adding an explanation.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx [hunk 1]
```diff
@@ -269,6 +269,12 @@ First you will want to [invoke your Lambda function](https://docs.aws.amazon.com
 
 Your telemetry will not appear under New Relic Serverless. Instead, you will find your telemetry data under the New Relic OpenTelemetry Nerdlets.
 
+## Distributed Tracing
+
+In some cases you may see fragmented distributed traces within New Relic with this configuration. This occurs when a trace starts or involves a service that is outside the ADOT context. When this happens, trace spans are usually created by AWS X-Ray and ADOT is not aware of them beyond a trace context being passed to it. This most commonly happens with managed AWS services that ADOT may interact, but that can only be traced using AWS X-Ray.
```
Comment (erabug) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx:274]:
```suggestion
In some cases you may see fragmented distributed traces within New Relic with this configuration. This occurs when a trace starts or involves a service that is outside the ADOT context (for example, a managed AWS service). Spans from that service are created by X-Ray, not OpenTelemetry, and ADOT does not forward them to New Relic.
```

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx [hunk 2]
```diff
@@ -269,6 +269,12 @@ First you will want to [invoke your Lambda function](https://docs.aws.amazon.com
...
+
+New Relic has an [X-Ray integration](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-x-ray-monitoring-integration/) to capture these spans created by X-Ray outside of an ADOT context. However, it has not yet been updated to support OpenTelemetry. Once this integration is updated, you should be able to see a full distributed trace within New Relic using OpenTelemetry.
```
Comment (erabug) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx:276]: I don't think we should mention the X-Ray integration at all, unless it's to recommend using it without any OpenTelemetry configuration if the fragmented traces are a real blocker for you.

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx (hunk 1 above)
Comment (erabug) [reply src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx:274]: I'd also add something like "Although the traces appear fragmented, they still provide complete insight into performance within the serverless function as well as other services whose spans were forwarded to New Relic."

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-java.mdx [hunk 3]
```diff
@@ -240,6 +240,12 @@ You can see an [example with OKHttp on GitHub](https://github.com/open-telemetry
   Refer to the [latest ARNs published by AWS](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) to verify the layer ARNs above are up to date.
 </Callout>
 
+## Distributed Tracing
```
Comment (erabug) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-java.mdx:243]: My recommendations also apply here of course!

**Review by erabug — CHANGES_REQUESTED** [review]
Thanks for opening this!

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx (hunk 2 above)
Comment (kolanos) [reply src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx:276]: Without that this is pretty much a dead end.

**Review by barbnewrelic — APPROVED** [review]
Thanks for providing good clarity for customers!

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx (hunk 2 above)
Comment (erabug) [reply src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx]: I totally agree we should prioritize the solution to forward and view _all_ AWS spans in New Relic. But as a general principle I don't think we should use language in the docs that promises particular implementations.

**Review by erabug — APPROVED** [review]

**Comment by barbnewrelic** [comment]
@kolanos  @erabug  Just checking in: Have you resolved your conversations about the wording in this draft? OK for tech writing team to proceed with publishing or canceling this request? Absolutely no rush, I just wanted to make sure I'm not a bottleneck to you. Thanks for confirming either way!

**Comment by erabug** [comment]
Yep, I approved the PR.

**Comment by barbnewrelic** [comment]
The build failed, so I'm going to try to jumpstart it again by closing and reopening your PR.

**Review by barbnewrelic — APPROVED** [review]
This time the build worked! I'll merge your PR now. Thanks again!

**Pull request merged**